app/views/reservations/new.html.erb

```diff
@@ -1,3 +1,6 @@
+<h2><%= @screening.movie.title %></h2>
+<p><%= "#{@screening.start_time.strftime("%d/%m/%Y %H:%M")}" %></p>
```

Comment:
use a helper for formatting time to make sure time formatting is consistant across the application